### PR TITLE
Add "leaf sorting"; reorganize side panel a bit

### LINKED ISSUE #170: Support iTOL-style "leaf sorting"
This would sort clades in the rectangular and circular layouts by the number of descendant leaves (see the iTOL docs [here](https://itol.embl.de/help.cgi)). I've seen this strategy used in lots of tree figures in papers, and it looks pretty :), so I assume it would be nice if this was an option. (Would be easiest to implement on the command line, but be useful to have in the browser.) 

Currently, Empress doesn't do this kind of sorting -- so the original order of clades in the tree's Newick file is followed.

**update:** I've implemented this in the `leafsorting` branch of my fork (https://github.com/fedarko/empress/tree/leafsorting). I imagine it would be easiest to revisit this once the current prototype branch is finalized and merged in.

### LINKED ISSUE #379: Reposition some controls in the side panel
From discussion with @ElDeveloper this morning: should move "show node circles", "ignore branch lengths," etc. to within the Layout tab. There are probably some other small ways to make things more intuitive, as well.

### PR DESCRIPTION
**Pull request by fedarko**
- Closes #170: adds leaf sorting, and changes the default way the tree is drawn to use `descending` leaf sorting (analogous to iTOL's default). It looks pretty nice :)

- Closes #379: the leaf sorting (and, now, ignore lengths) options are now located in the `Layout` tab of the side panel.

stuff

It is probably worth noting that changing the default to use leaf sorting makes basically all of the README screenshots slightly out-of-date ... it might be worth seeing if we can finagle McHelper to recompute the README screenshots for every commit? (For now I don't think it's a big deal, though.)

### CODE REVIEW

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empire-biplot.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F394%2Fempire-biplot.qzv), [empire.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F394%2Fempire.qzv), [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F394%2Fempress-tree.qzv)

**Review by ElDeveloper — APPROVED** [review]
Looks great, thanks!

File: tests/test-empress.js [hunk 1]
```diff
@@ -517,46 +517,29 @@ require(["jquery", "UtilitiesForTesting", "util", "chroma"], function (
...
-            ok(
-                Math.abs(this.empress.layoutAvgPoint.Rectangular[0] - 7) <=
-                    1.0e-15
-            );
-            // y coor for rectangular layout
-            ok(
-                Math.abs(this.empress.layoutAvgPoint.Rectangular[1] - 8) <=
-                    1.0e-15
-            );
+            ok(Math.abs(avgPt[0] - 7) <= 1.0e-15);
```
[the remark this answers is not in the document]
Comment (fedarko) [reply tests/test-empress.js]: Yep! Just made that change.

**Comment by ElDeveloper** [comment]
Thanks @fedarko

**Pull request merged**